src/generate_trigger.ts

```diff
@@ -0,0 +1,112 @@
+import { parse } from "./deps.ts";
+
+export type WorkflowRecord = {
+  "callback_id": string;
+};
+
+interface Inputs {
+  [key: string]: { value: string };
+}
+
+export type TriggerFileRecord = {
+  "type": string;
+  "name": string;
+  "description"?: string;
+  "workflow": WorkflowRecord;
+  "inputs"?: Inputs;
+};
+
+export type TriggersPayload = {
+  triggers: TriggerFileRecord[];
+};
+
+export interface Response {
```

Comment:
If this interface will not be used outside of this file in this repo, let's not export it. Otherwise, let's name it more specifically.